src/main/java/frc/robot/subsystems/VisionSubsystem.java

```diff
@@ -25,147 +22,88 @@
 
   // Deviations
   private static final Vector<N3> STANDARD_DEVS =
       VecBuilder.fill(0.1, 0.1, Units.degreesToRadians(20));
   private static final Vector<N3> DISTANCE_SC_STANDARD_DEVS =
       VecBuilder.fill(1, 1, Units.degreesToRadians(50));
 
-  // making the cameras, pose estimator, field2d, fieldObject2d, april tags helper objects
-  private final PhotonCamera leftCamera;
-  private final PhotonCamera rightCamera;
-  private final PhotonPoseEstimator photonPoseEstimatorLeftCamera;
-  private final PhotonPoseEstimator photonPoseEstimatorRightCamera;
-  private final Field2d robotField;
-  private final FieldObject2d rawVisionFieldObject;
-  private final DrivebaseWrapper aprilTagsHelper;
+  // AprilTag field layout for 2025
+  private static final AprilTagFieldLayout fieldLayout =
+      AprilTagFieldLayout.loadField(AprilTagFields.k2025ReefscapeWelded);
 
-  // These are always set with every pipeline result
-  private PhotonPipelineResult latestResult = null;
+  private final DrivebaseWrapper aprilTagsHelper;
 
-  // These are only set when there's a valid pose & last timestamps, last poses and distance
-  // creation
-  private double lastTimestampSeconds = 0;
-  private double lastRawTimestampSeconds = 0;
-  private Pose2d lastFieldPose = new Pose2d(-1, -1, new Rotation2d());
-  private double Distance = 0;
+  private final Field2d robotField;
+  private final FieldObject2d rawVisionFieldObject;
 
-  // boolean to disable vision
-  // if you press the button on shuffleboard it disables vision
   private final GenericSubscriber disableVision;
+  private final LLCamera leftCamera = new LLCamera(LIMELIGHT_LEFT);
+  private final LLCamera rightCamera = new LLCamera(LIMELIGHT_RIGHT);
 
-  // full field pose for logging
   private final StructPublisher<Pose3d> fieldPose3dEntry =
       NetworkTableInstance.getDefault()
           .getStructTopic("vision/fieldPose3d", Pose3d.struct)
           .publish();
-  // left camera field pose for logging
   private final StructPublisher<Pose3d> rawFieldPose3dEntryLeft =
       NetworkTableInstance.getDefault()
-          .getStructTopic("vision/rawFieldPose3dLeft", Pose3d.struct)
+          .getStructTopic("vision/rawFieldPose2dLeft", Pose3d.struct)
           .publish();
-  // right camera field pose for logging
   private final StructPublisher<Pose3d> rawFieldPose3dEntryRight =
       NetworkTableInstance.getDefault()
           .getStructTopic("vision/rawFieldPose3dRight", Pose3d.struct)
           .publish();
-  // field map for bot to april tag (configured for 2025)
-  private static final AprilTagFieldLayout fieldLayout =
-      AprilTagFieldLayout.loadField(AprilTagFields.k2025ReefscapeWelded);
 
-  // method creation for subsystem init in subsystems.java
+  // state
+  private final double lastTimestampSeconds = 0;
+  private double lastRawTimestampSeconds = 0;
+  private Pose2d lastFieldPose = new Pose2d(-1, -1, new Rotation2d());
+  private double Distance = 0;
```

Comment:
`Distance` should have the same casing style as the other class variables. And it should clarify what the distance is between & when it was measured.